Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -58,6 +58,7 @@
 static const float s_kCGGradientOffsetPoint = 1E-45;
 
 enum _CGCoordinateMode : unsigned int { _kCGCoordinateModeDeviceSpace = 0, _kCGCoordinateModeUserSpace };
+enum _CGAntialiasToggle : unsigned int { _kCGToggleDefault = 2, _kCGToggleOff = 0, _kCGToggleOn = 1 };
```

Comment:
The point of a default value here is to avoid setting any Anti aliasing information when we don't have to. It avoids the work of updating the D2D1 drawing parameters when nothing has been changed and allows D2D to make the decisions as to the best AA to use.